docs/setup.asciidoc

```diff
@@ -361,6 +366,25 @@ connection.UseElasticApm();
 A callback is registered with the `IConnectionMultiplexer` to provide a profiling session for each transaction and span that captures redis commands
 sent with `IConnectionMultiplexer`.
 
+[[setup-mongo-db]]
+=== MongoDB.Driver
+
+[float]
+==== Quick start
+
+A prerequisite for auto instrumentation with [`MongoDb.Driver`] is to configure the `MongoClient` with `MongoDbEventSubscriber`:
+
+[source,csharp]
+----
+var settings = MongoClientSettings.FromConnectionString(mongoConnectionString);
+
+settings.ClusterConfigurator = builder => builder.Subscribe(new MongoEventSubscriber());
+var mongoClient = new MongoClient(settings);
+----
+
+If the agent is included by referencing the `Elastic.Apm.NetCoreAll` package, it will automatically capture calls to MongoDB on every active transaction.
```

Comment:
```suggestion
With the above configuration in place, If the agent is included by referencing the `Elastic.Apm.NetCoreAll` package, it will automatically capture calls to MongoDB on every active transaction.
```

I think this paragraph should be connected with the code sample, so that it can't be misinterpreted in isolation

Reply:
Smart. What do you think about changing the wording just a hair?

```suggestion
Once the above configuration is in place, and if the agent is included by referencing the `Elastic.Apm.NetCoreAll` package, it will automatically capture calls to MongoDB on every active transaction.
```

Reply:
LGTM, thanks @bmorelli25 😃

Reply:
:) updated.